server/main.go

```diff
@@ -73,6 +76,28 @@ func main() {
 	prometheus.MustRegister(exporter, metricExporter)
 	mux.Handle("/metrics", promhttp.Handler())
 
-	err := http.ListenAndServe(cli.HTTPAddress, mux)
-	ctx.FatalIfErrorf(err)
+	go func() {
+		err := http.ListenAndServe(cli.HTTPAddress, mux)
```

Comment:
Isn't it enough to just move it to the end of function? Then you should not need to run it in separate goroutine.

Reply:
AFAIK ListenAndServe runs indefinitely. Meaning the anything after it doesn't get executed. However it should block main function from exiting. Rearranging the ticker loop and removing separate goroutine around ListenAndServe